### DISCUSSION TITLE
-----------------

### DISCUSSION THREAD

**Discussion by Toso82** [SixLabors/ImageSharp.Web#184] [Q&A] [answered]

**Comment by JimBobSquarePants** [comment, marked as answer]
Wrong repository mate

**Reply by JimBobSquarePants** [reply]
Wrong library. This is ImageSharp.Web.

**Comment by Toso82** [comment]


**Reply by JimBobSquarePants** [reply]
Please stop spamming this repo with code from other libraries

**Comment by NikRimington** [comment]
@Toso82 , you are posting questions about ImageProcessor functionality (https://github.com/JimBobSquarePants/ImageProcessor) on this repository which is about ImageSharp.

Image Sharp is **NOT** a rewrite of ImageProcessor for .Net  Core, it is a completely new library that works in a different way, it is not "a new version of ImageProcessor". The services and functionality you are asking for is not how ImageSharp works.

If you want to try and replicate this functionality for your project, I would advise looking at how ImageProcessor works, then seeing how you can create you own "RemoteImageService" that wraps around ImageSharp, but that would be entirely on yourself and wouldn't be the responsibility of this incredible team.
